<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindowClass</class>
 <widget class="QMainWindow" name="EditorWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2664</width>
    <height>1797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nova-Editor</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="nova::editor::EngineWidget" name="widget_engine" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2664</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu_file"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dock_widget_debug">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Debug</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>200</height>
     </size>
    </property>
    <widget class="QTextEdit" name="text_edit_debug">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>2664</width>
       <height>191</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>nova::editor::EngineWidget</class>
   <extends>QWidget</extends>
   <header location="global">Editor\Engine\Widgets\EngineWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="EditorWindow.qrc"/>
 </resources>
 <connections/>
</ui>
